draft-ietf-httpbis-origin-frame.md

```diff
@@ -72,6 +72,18 @@ as described in BCP 14 {{!RFC2119}} {{!RFC8174}} when, and only when, they appea
 capitals, as shown here.
 
 
+# The ORIGIN_SUPPORT HTTP/2 Setting {#setting}
+
+The ORIGIN_SUPPORT HTTP/2 setting {{!RFC7540}}, Section 6.5) indicate that a server
+desires the connection to be considered "opted in" to the behaviour described in this specification.
+
+The value of the ORIGIN_SUPPORT setting does not have any meaning. Note that support for this
+specification cannot be disabled once it is sent.
```

Comment:
We've used default = 0 and supported = 1 in the past.  I think that I'd prefer that.

Reply:
That will cause people to assume they can turn it off; is it worth the added complexity / misunderstanding?

Reply:
Hmm, that's nasty.  I don't know.  Consistency is valuable though.

How about you mandate a value of 1.  Anything else is an error.

Reply:
In HTTP/QUIC, it can be a Boolean (sent = true), but HTTP/2 has to have a value.  I'm on the mandate-value-of-1 wagon.